_sass/catalog.scss

```diff
@@ -455,12 +455,11 @@ input[type="checkbox"] {
   }
 }
 .modal-pattern-name {
-  color: #333;
-  text-align: center;
+color: #333;
+  margin-left: 40px;
```

Comment:
- add indentation to color like before
- remove margin-left as it is affecting the title positioning 
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/54144759/212131993-937cc1c7-6c74-45d2-9077-762dcb63e105.png">